rotkehlchen/data_import/importers/binance.py

```diff
@@ -434,6 +550,72 @@ def process_entry(
         importer.add_history_events(write_cursor=write_cursor, history_events=[staking_event])
 
 
+class BinanceUSDMProgram(BinanceSingleEntry):
+    """All Binance USDM Program Single Events are processed here"""
+    ACCOUNT = 'USD-MFutures'  # specifying account because fee is ambiguous
+    AVAILABLE_OPERATIONS = (
+        'Fee',
+        'Funding Fee',
+        'Realized Profit and Loss',
+    )
+
+    def is_entry(self, requested_operation: str, account: str, change: int) -> bool:
+        if requested_operation in ('Fee', 'Funding Fee') and change == abs(change):
+            return False
+        return requested_operation in self.AVAILABLE_OPERATIONS and account == self.ACCOUNT
+
+    def _get_event(self, timestamp: Timestamp, data: BinanceCsvRow) -> HistoryEvent:
+        """
+        Process USDM Program Events. It includes
+        Fee, Funding Fee, and Realized Profit and Loss. May raise:
+        - KeyError
+        """
+        event_identifier = f'{EVENT_IDENTIFIER_PREFIX}{hash_csv_row(data)}'
+        timestamp_ms = ts_sec_to_ms(timestamp)
+        amount = abs(data['Change'])
+        asset: AssetWithOracles = data['Coin']
+
+        if data['Operation'] in ('Fee', 'Funding Fee'):
+            event_type = HistoryEventType.SPEND
+            event_subtype = HistoryEventSubType.FEE
+            notes = f'{amount} {asset.symbol} fee paid on binance USD-MFutures'
+        else:  # is Realized Profit and Loss
+            is_profit = data['Change'] == amount
+            event_type = HistoryEventType.RECEIVE if is_profit else HistoryEventType.SPEND
+            event_subtype = HistoryEventSubType.REWARD if is_profit else HistoryEventSubType.PAYBACK_DEBT  # noqa: E501
+            action = 'profit' if is_profit else 'loss'
+            notes = f'{amount} {asset.symbol} realized {action} on binance USD-MFutures'
+        history_event = HistoryEvent(
+            event_identifier=event_identifier,
+            sequence_index=0,
+            timestamp=timestamp_ms,
+            location=Location.BINANCE,
+            event_type=event_type,
+            event_subtype=event_subtype,
+            asset=data['Coin'],
+            balance=Balance(amount=amount),
+            location_label='CSV import',
+            notes=notes,
+        )
+        return history_event
```

Comment:
don't create temporary variables that are used only in one place

```suggestion
        return HistoryEvent(
            event_identifier=event_identifier,
            sequence_index=0,
            timestamp=timestamp_ms,
            location=Location.BINANCE,
            event_type=event_type,
            event_subtype=event_subtype,
            asset=data['Coin'],
            balance=Balance(amount=amount),
            location_label='CSV import',
            notes=notes,
        )
```